From ef038a3a4f09657e4dbcfd47a145c87895896147 Mon Sep 17 00:00:00 2001
From: Kent Fredric <kentnl@gentoo.org>
Date: Sun, 8 Jul 2018 16:06:05 +1200
Subject: [PATCH 21/28] Disable porting tests which create fun false-failures
 all over travis

This is due to our patches tweaking code in-band, and this test
deciding we need to increment the version as a result.
---
 t/porting/cmp_version.t | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/t/porting/cmp_version.t b/t/porting/cmp_version.t
index 0573e2f81d..f5738c5837 100644
--- a/t/porting/cmp_version.t
+++ b/t/porting/cmp_version.t
@@ -16,7 +16,12 @@
 # same as that commit, report it.
 #
 # There's also a module exclusion list in Porting/cmpVERSION.pl.
-
+BEGIN {
+  if ( $ENV{GENTOO_NO_PORTING_TESTS} ) {
+    print "1..0 # SKIP GENTOO_NO_PORTING_TESTS set";
+    exit;
+  }
+}
 BEGIN {
     @INC = '..' if -f '../TestInit.pm';
     @INC = '.'  if -f  './TestInit.pm';
-- 
2.39.3

